CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Use native scrollbars on macOS, iOS and Android
 - Fix address signing fail on screen rotation for Pocket and Bitsurance
 - Restrict selection to text files when importing notes
+- Remove hide amounts settings
```

Comment:
Nit: if I read this it would be unclear to me if this feature is removed or just the setting.
How about something like permanently enable hide amount feature or similar?